src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/4.diff.sql

```diff
@@ -447,6 +477,132 @@ AS
     COMMIT TRANSACTION
 GO
 
+/***************************************************************************************/
+-- STORED PROCEDURE
+--     GetExtendedQueryTagErrors
+--
+-- DESCRIPTION
+--     Gets the extended query tag errors by tag path.
+--
+-- PARAMETERS
+--     @tagPath
+--         * The TagPath for the extended query tag for which we retrieve error(s).
+--
+-- RETURN VALUE
+--     The tag error fields and the corresponding instance UIDs.
+/***************************************************************************************/
+CREATE OR ALTER PROCEDURE dbo.GetExtendedQueryTagErrors (@tagPath VARCHAR(64))
+AS
+    SET NOCOUNT     ON
+    SET XACT_ABORT  ON
+    BEGIN TRANSACTION
+
+        DECLARE @tagKey     INT
+        SELECT @tagKey = TagKey
+        FROM dbo.ExtendedQueryTag WITH(HOLDLOCK)
+        WHERE dbo.ExtendedQueryTag.TagPath = @tagPath
+
+        -- Check existence
+        IF (@@ROWCOUNT = 0)
+            THROW 50404, 'extended query tag not found', 1 
+
+        SELECT
+            TagKey,
+            ErrorCode,
+            CreatedTime,
+            StudyInstanceUid,
+            SeriesInstanceUid,
+            SopInstanceUid
+        FROM dbo.ExtendedQueryTagError AS XQTE
+        INNER JOIN dbo.Instance AS instance ON XQTE.Watermark = instance.Watermark
```

Comment:
nit: in terms of style, we typically make the alias an abbreviation, like in this case `I`